### PR TITLE
Adds Node 16 to CI

### PR DESCRIPTION
**Pull request by michaelgoin**
## Proposed Release Notes

* Added Node 16 to CI.

## Links

* Resolves: https://github.com/newrelic/node-newrelic/issues/756
* https://github.com/newrelic/node-newrelic/issues/771

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,8 +122,8 @@
     "ca-gen": "./bin/update-ca-bundle.sh",
     "docker-env": "./bin/docker-env-vars.sh",
     "docs": "npm ci && jsdoc -c ./jsdoc-conf.json --private -r .",
-    "integration": "npm run prepare-test && npm run sub-install && time tap test/integration/**/**/*.tap.js --timeout=180 --no-coverage",
-    "prepare-test": "npm run ca-gen && npm run ssl && npm run docker-env",
+    "integration": "npm run prepare-test && npm run sub-install && time tap test/integration/**/**/*.tap.js --timeout=180 --no-coverage --reporter classic",
+    "prepare-test": "npm ci && npm run ca-gen && npm run ssl && npm run docker-env",
```
Comment (carlo-808) [inline package.json:126]: Do we need `npm ci` here in prepare-test?

Comment (bizob2828) [reply package.json:126]: I don't think so. I removed this a few weeks ago because in CI we run npm install before calling any other scripts

Comment (bizob2828) [reply package.json:126]: i will fix this and then approve

**Review by bizob2828 — APPROVED** [review]

**Comment by bizob2828** [comment]
Closes #803

**Pull request merged**